cmd/libs/go2idl/client-gen/generators/normalization/normalization.go

```diff
@@ -29,3 +36,50 @@ func Version(version string) string {
 	}
 	return version
 }
+
+func GroupVersion(gv unversioned.GroupVersion) unversioned.GroupVersion {
+	return unversioned.GroupVersion{Group(gv.Group), Version(gv.Version)}
+}
+
+// Our assumption is that for the types.go that includes the internal version,
+// it's either located at.../pkg/api/ or .../pkg/apis/<group>/;
```

Comment:
We will need to make this work with additional source directories. I.e., the `pkg/` part of those paths will need to be parameterizable. Eventually.